string env_name
---
bool success
string[] plugins
string error
